Source/Boxed.AspNetCore.TagHelpers/OpenGraph/ObjectTypes/OpenGraphMetadata.cs

```diff
@@ -359,13 +361,11 @@ protected virtual void Validate()
         private string GetRequestUrl()
         {
             var httpContext = this.ViewContext.HttpContext;
-            var urlHelper = httpContext
-                .Features
-                .Get<IServiceProvidersFeature>()
-                .RequestServices
-                .GetRequiredService<IUrlHelper>();
+            var urlHelper = httpContext.GetUrlHelper();
             var request = httpContext.Request;
-            return new Uri(new Uri(request.Scheme + "://" + request.Host.Value), urlHelper.Content(request.Path.Value)).ToString();
+            var uriBuilder = new UriBuilder(request.Scheme, request.Host.Value);
```

Comment:
You are not using the `uriBuilder` variable in the return. Using it is a great idea.